### DISCUSSION TITLE
Welcome to eslint-stylistic Discussions!

### DISCUSSION THREAD

**Discussion by antfu** [eslint-stylistic/eslint-stylistic#33] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by alexleduc76** [comment]
Just wanted to say thanks, for maintaining this part of ESLint my team uses in all the code bases we work on, to make them optimally readable.
